account_financial_report/report/general_ledger.py

```diff
@@ -20,7 +20,8 @@ def _get_analytic_data(self, account_ids):
         analytic_accounts = self.env["account.analytic.account"].browse(account_ids)
         analytic_data = {}
         for account in analytic_accounts:
-            analytic_data.update({account.id: {"name": account.name}})
+            name = f"{account.code} - {account.name}" if account.code else account.name
```

Comment:
For following the display name:

```suggestion
            name = f"[{account.code}] {account.name}" if account.code else account.name
```

Reply:
Done!